browser/ui/brave_actions/brave_action_icon_with_badge_image_source.cc

```diff
@@ -22,6 +22,19 @@
 #include "ui/gfx/image/image_skia_operations.h"
 #include "ui/gfx/skia_paint_util.h"
 
+namespace {
+// Always use same height to avoid jumping up and down with different
+// characters which will differ slightly,
+// but vary the width so we cover as little of the icon as possible.
+constexpr int kBadgeHeight = 12;
+constexpr int kBadgeMaxWidth = 16;
+constexpr int kVPadding = 1;
+constexpr int kVMarginTop = 2;
+constexpr int kOuterCornerRadius = 5;
+constexpr int kTextHeightTarget = kBadgeHeight - (kVPadding * 2);
+constexpr int kMaxIncrementAttempts = 5;
```

Comment:
This is fine, but the reason they were put inline in the function (still as `constexpr`) was because they only needed to be scoped to 1 function. Where they are now moved, they could be scoped to multiple functions even though that's not neccessary, and some of them aren't named specifically for that one function's purpose, e.g. `kMaxIncrementAttempts` which applies to just 1 loop in 1 function.
But since this file is 90% taken up by that function, I'll not argue too much against it!